task1.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+
+def devide_by_number(inpt, dev):
+    if inpt % dev == 0:
+        return True
+
+
+for i in range(1, 101):
+    if devide_by_number(i, 15):
+        print "fizzbizz"
+        continue
```

Comment:
remove redundant `continue`

Reply:
if we continue loop, it will print "fizz" and "buzz" because this number also divided by 3 and 5

Reply:
so use `elif/else` constructions,
`continue` isn't best practice